packages/react-native-win32/package.json

```diff
@@ -54,8 +54,10 @@
     "jscodeshift": "^0.6.2",
     "just-scripts": "^0.36.1",
     "react": "16.9.0",
+    "react-native-svg-transformer": "^0.14.3",
     "react-native-windows-override-tools": "^0.0.1",
     "react-native": "0.61.5",
+    "react-native-svg": "^11.0.0",
```

Comment:
Have you tested this against the current build of react-native-win32 with version number of `0.0.0-master.x`? I would expect you need a selective dependency resolution to force this, unless the peer deps here are super permissive.

Either way, @acoates-ms brought up the conundrum where it's likely this will break as we rev our repo and breaking changes happen upstream before third party deps are updated. It's questionable whether we should have dependencies on community modules.

Reply:
From react-native-svg:

```json
  "peerDependencies": {
    "react": "*",
    "react-native": ">=0.50.0"
  },
```

This will break once we're using React Native master builds which are versioned as `react-native@0.0.0-xxx`. This is a bit of a sign we shouldn't have this dependency in this package.